src/components/tab-title/tab-title.scss

```diff
@@ -7,7 +7,9 @@
 :host([layout="center"]) {
   @apply my-0 mx-5 text-center;
   flex-basis: theme("spacing.48");
-  margin: auto;
+  .content {
+    @apply m-auto;
```

Comment:
given that there's no difference between `margin: auto` and `@apply m-auto` I'd recommend using `margin: auto` it's more readable

Reply:
Since we're using tailwind for now, it makes sense to stick to tailwind classes if they are available, for consistency. This is what we've been doing so far. Also, overlapping uses of tailwind and regular css props might create oversight doubling use of both by accident.

Reply:
Agreed, lets keep tailwind for now for consistency. Hopefully we can remove tailwind in the future.